ocaml/util/gen_build_info_ml.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -e
+
+DATE=$(date "+%Y-%m-%d")
+# shellcheck disable=SC2001
+XAPI_VERSION=$(echo "$XAPI_VERSION" | sed "s/^v//g")
```

Comment:
* Should this fail when there is no XAPI_VERSION defined? 
* The substitution can be done like this `XAPI_VERSION=${XAPI_VERSION#v}`